rotkehlchen/db/filtering.py

```diff
@@ -1112,6 +1112,7 @@ def make(
             entry_types: Optional[list[HistoryBaseEntryType]] = None,
             exclude_ignored_assets: bool = False,
             validator_indices: Optional[list[int]] = None,
+            location_labels_or_validator_indices: Optional[tuple[list[str], list[int]]] = None,
```

Comment:
Hmm okay this approach is quite bad.

We do not have such logic in the filters. Yes indeed there is no "OR"ing filters but this is not the way to go. This is a very ugly hack.

Why do you need to or those? As I wrote in the api.rest file I don't think this is needed at all